README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is markup that means something. You can't just have random stuff and expect people to know what it's for so you have to make sense.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+block takes up the full width and inline only takes the space that it needs. inline respects height, width, margin and padding and block does not.
+
 3. What are the 4 areas of the box model?
 
+margin, border, padding, content.
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+cross axis.
+
 5. Explain why git is valuable to a team of developers.
 
+It allows for collaberation from remote areas.
```

Comment:
What other benefits are there to working with git?

Reply:
Branching and merging are easy, it has code repository, its secure and has a flexible workflow.